RecoCTPPS/Configuration/python/recoCTPPS_sequences_cff.py

```diff
@@ -9,6 +9,11 @@
 from Geometry.VeryForwardGeometryBuilder.ctppsIncludeAlignmentsFromXML_cfi import *
 ctppsIncludeAlignmentsFromXML.RealFiles = cms.vstring("Alignment/CTPPS/data/RPixGeometryCorrections.xml")
 
+#--- temporary workaround while calibration conditions are integrated within the DB
+from CondFormats.CTPPSReadoutObjects.ppsTimingCalibrationESSource_cfi import ppsTimingCalibrationESSource
+ppsTimingCalibrationESSource.calibrationFile = cms.FileInPath('RecoCTPPS/TotemRPLocal/data/timing_offsets_ufsd_2018.dec18.cal.json')
```

Comment:
similar to what's in L10, this approach of loading a default config and modifying it is dangerous.
The relevant configs for the ES sources should be defined in a single place where such sources are defined.
Any other client of these ES sources is likely to get inconsistent setup.